src/plugins/VideoRoomPlugin.ts

```diff
@@ -129,6 +139,18 @@ export class VideoRoomPlugin extends BasePlugin {
     if (pluginData?.videoroom === 'joined') {
       this.onPublisherInitialStateUpdate(msg)
     }
+
+    if (msg.janus === 'slowlink') {
```

Comment:
Can't we do this in the same if `if (msg.janus === 'slowlink' && msg.uplink)`  ?

Reply:
Actually not. We need different processing for uplink and downlink. Uplink (which means publisher) requires cutting bitrate on publishing stream, but downlink (subscriber) requires sending message to janus with request to cut bitrate of the guy we have issues to subscribe to. 
This logic needs to be adjusted when simulcasting will be available. Uplink handling should stay almost the same (but instead of cutting bitrate we can cancel publishing top quality layer). For downlink instead of asking to reduce bitrate another publisher we will be able just to switch to lower quality layer